Scripts/Level_Components/Level_Elements/LevelControl.gd

```diff
@@ -40,3 +42,19 @@ func _on_DoubleSpeed_toggled(button_pressed):
 		GameStats.run_speed = 0.25
 	else:
 		GameStats.run_speed = 0.50
+
+
+func _on_Help_pressed():
+	#If there isn't anymore dialogue to display
+	if DialogueManager.dialogue_queue.size() == 0:
+		var level = get_node("..")
+		DialogueManager.restart_dialogue()
+		if level.has_tutorial:
```

Comment:
Rewrite logic a bit since some levels might not have a tutorial but will still have Robby say something at the beginning and the end of the level.